frontend/src/app/HealthChecks.tsx

```diff
@@ -1,12 +1,15 @@
 import { Route, Routes } from "react-router-dom";
 
+import ProdSmokeTest from "./ProdSmokeTest";
+
 const HealthChecks = () => (
   <Routes>
     <Route path="ping" element={<div>pong</div>} />
     <Route
       path="commit"
       element={<div>{process.env.REACT_APP_CURRENT_COMMIT}</div>}
     />
+    <Route path="prod-smoke-test" element={<ProdSmokeTest />} />
```

Comment:
Its pedantic, but it's not a prod-specific smoke test.  I would use something generic like `health` for the route and the component name